corehq/apps/app_manager/static/app_manager/js/app-notifications.js

```diff
@@ -0,0 +1,24 @@
+/* globals hqDefine moment */
+hqDefine('app_manager/js/app-notifications.js', function () {
+    function NotifyFunction(userId, $element) {
+        return function(msg) {
+            var msgObj = JSON.parse(msg);
+            // only show notifcations from other users
+            if (msgObj.user_id !== userId) {
+                var $inner;
+                // if nothing in the outer element, add the notification div and populate
```

Comment:
This is pretty similar to what [notify_user](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/style/templates/style/base.html#L399-L407) does, can you use that instead?